src/utils/browser.js

```diff
@@ -17,9 +17,9 @@ const Browser = (function Browser() {
   return {
     isSafari: isSafari(),
     isUiWebView: /(iPhone|iPod|iPad).*AppleWebKit(?!.*Safari)/i.test(window.navigator.userAgent),
-    ie: window.navigator.pointerEnabled || window.navigator.msPointerEnabled,
+    ie: window.navigator.pointerEnabled || window.PointerEvent || window.navigator.msPointerEnabled,
```

Comment:
What's the `ie` check for? If it's just for pointer events maybe the detect should be renamed as other browsers support it too.

Reply:
I guee it is a check for IE / Edge to see if touch is supported. There are also other checks. But you are right, this is partly browser sniffing.